### PR TITLE
Upgraded Storybook and implemented some storybook addons

### PR DESCRIPTION
**Pull request by pppemo**

### CODE REVIEW

File: packages/react/.babelrc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "exclude": "node_modules/**",
-  "plugins": [],
+  "plugins": ["@babel/plugin-proposal-class-properties"],
```
Comment (pppemo) [inline packages/react/.babelrc]: Eslint was complaining to some arrows when this wasn't there

File: packages/react/.storybook/config.js [hunk 2]
```diff
@@ -9,4 +10,47 @@ function loadStories() {
...
+addParameters({
+  options: {
+    name: "Heetch Flamingo",
+    addonPanelInRight: true,
+    hierarchySeparator: /\//,
+  },
+});
+
+const flamingoViewports = {
+  BreakpointS: {
```
Comment (ugogo) [inline packages/react/.storybook/config.js]: Is this documented somewhere?
Shouldn't we use `mobile/tablet/desktop`?

File: packages/react/.babelrc (hunk 1 above)
Comment (ugogo) [inline packages/react/.babelrc]: Why is this needed?

Comment (pppemo) [reply packages/react/.babelrc]: Eslint was complaining to some arrows when this wasn't there
Don't have a clear understanding why though...

File: packages/react/.storybook/config.js (hunk 2 above)
Comment (pppemo) [reply packages/react/.storybook/config.js]: https://www.notion.so/heetch/Grid-1e18d12216b040c28d0e146125e7fded#dab7afa3ba2947f6a45a98e59d89d56a

File: packages/react/.babelrc (hunk 1 above)
Comment (ugogo) [reply packages/react/.babelrc]: Weird, okay then 👍🏼

File: packages/react/.storybook/config.js (hunk 2 above)
Comment (ugogo) [reply packages/react/.storybook/config.js]: 👍
